apps/play/src/SIMPLIFICATION.md

```
@@ -0,0 +1,29 @@
+# Recommended simplification order:
+
+### Remove the duplicate sampler engine first — high value / low effort.
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)